### PR TITLE
Rat story

### PR DESCRIPTION
**Pull request by Rambo001**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/light-fire/dream/dream.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+But wait. The rat starts growing in size.
+
+You better look scared. 
+
+The dark place has some light now.Not the spotlight!!
+
+A fire ring around your battlefield. Yes,a battlefield!! 
+
+[Start fighting](fight/fight.md)
```
Comment (casey-collab) [inline english/light-fire/dream/dream.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/light-fire/dream/dream.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+[Yes you don't have an option!!!Start fighting!!!](fight/fight.md)
```
Comment (casey-collab) [inline english/light-fire/dream/dream.md]: This link is also invalid.

File: english/light-fire/fire/alone.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+So you started screaming seeing a rat!!!
+
+Well you just forgot about something in the meantime...
+
+The room is well lit-up.
+
+[Try to extinguish the fire](die/die.md)
```
Comment (casey-collab) [inline english/light-fire/fire/alone.md]: This link is also invalid.

File: english/light-fire/fire/alone.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+[Enter that opening](magic-world/alice.md)
```
Comment (casey-collab) [inline english/light-fire/fire/alone.md]: This link is also invalid.

File: english/light-fire/follow/suspense.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+You thought it was the way to wonderland did you??
+
+Guess what? You are wrong!!
+
+The rat finally goes into a dark place.
+
+When you look back, you see nothing...
+
+[Go back](die/die.md)
```
Comment (casey-collab) [inline english/light-fire/follow/suspense.md]: This link is also invalid.

File: english/light-fire/follow/suspense.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+[Go to that dark place](dream/dream.md)
```
Comment (casey-collab) [inline english/light-fire/follow/suspense.md]: This link is also invalid.

File: english/light-fire/magic-world/alice.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+When you follow the rat, it doesn't seem just like another hideout.
+
+It appeared to be a pathway of some sort.
+
+[You follow the rat](follow/suspense.md)
```
Comment (casey-collab) [inline english/light-fire/magic-world/alice.md]: This link is also invalid.

File: english/light-fire/magic-world/alice.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+
+[You go back](back/back.md)
```
Comment (casey-collab) [inline english/light-fire/magic-world/alice.md]: This link is also invalid.

File: english/light-fire/magic-world/alice.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+
+[You have already seen Alice in Wonderland](seen/seen.md)
```
Comment (casey-collab) [inline english/light-fire/magic-world/alice.md]: This link is also invalid.

File: english/light-fire/back/back.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+By this time, your marshmallow room has burnt down.
```
Comment (casey-collab) [inline english/light-fire/back/back.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

File: english/light-fire/follow/suspense.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+You thought it was the way to wonderland did you??
```
Comment (casey-collab) [inline english/light-fire/follow/suspense.md]: This page is also unreachable.

File: english/light-fire/seen/seen.md [hunk 12]
```diff
@@ -0,0 +1,5 @@
+So you thought this was about Alice in Wonderland.
```
Comment (casey-collab) [inline english/light-fire/seen/seen.md]: This page is also unreachable.

File: english/light-fire/dream/dream.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+You decided to move forward...
```
Comment (casey-collab) [inline english/light-fire/dream/dream.md]: This page is also unreachable.

File: english/light-fire/die/die.md [hunk 14]
```diff
@@ -0,0 +1 @@
+Well, it's simple. You Die. Rest in Peace.
```
Comment (casey-collab) [inline english/light-fire/die/die.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Rambo001** [comment]
Fixed the links

**Comment by Rambo001** [comment]
:)

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**